### PR TITLE
New LSP project

### PR DESCRIPTION
**Pull request by dianamarin93**
I added new LSP project.

### CODE REVIEW

File: Liskov Substitution Principle/Program.cs [hunk 1]
```diff
@@ -10,6 +10,21 @@ class Program
...
+            Console.WriteLine(car);
+            car.StartEngine();
+
+            MotorisedVehicle train = new Train("LDE", 80, 150, new Engine("LDE", 2100));
+            Console.WriteLine(train);
+            train.StartEngine();
+
+            UnmotorisedVehicle bicycle = new Bicycle("Pegas", 20, 1);
+            Console.WriteLine(bicycle);
+
```
Comment (GabrielCorlan) [inline Liskov Substitution Principle/Program.cs]: You should remove extra spaces

File: Liskov Substitution Principle/Vehicle.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace Liskov_Substitution_Principle
+{
+    class Vehicle
+    {
+        public string Name { get; set; }
+        public int Speed { get; set; }
+        public int MaxPassengers { get; set; }
+
+
```
Comment (GabrielCorlan) [inline Liskov Substitution Principle/Vehicle.cs]: same as above

File: Liskov Substitution Principle/Vehicle.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            Name = name;
+            Speed = speed;
+            MaxPassengers = maxPassengers;
+            
+        }
+
+        public override string ToString()
+            => $"Name : {Name}, Speed: {Speed}, MaxPassengers: {MaxPassengers}";
+
+
```
Comment (GabrielCorlan) [inline Liskov Substitution Principle/Vehicle.cs]: same

File: Liskov Substitution Principle/Vehicle.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        public int Speed { get; set; }
+        public int MaxPassengers { get; set; }
+
+
+        public Vehicle(string name, int speed, int maxPassengers)
+        {
+            Name = name;
+            Speed = speed;
+            MaxPassengers = maxPassengers;
+
```
Comment (GabrielCorlan) [inline Liskov Substitution Principle/Vehicle.cs]: same

**Review by GabrielCorlan — CHANGES_REQUESTED** [review]

File: Liskov Substitution Principle/Program.cs (hunk 1 above)
Comment (dianamarin93) [reply Liskov Substitution Principle/Program.cs]: Done!

**Review by GabrielCorlan — APPROVED** [review]
Looks great! Approved!

**Pull request merged**